<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chat</class>
 <widget class="QWidget" name="chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="chat_msg">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>281</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="input_edit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="send_btn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_btn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="friend_name">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>friends</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
